web/src/app/components/modal/modal.component.ts

```diff
@@ -182,19 +184,21 @@ export class ModalComponent extends BaseModal implements OnInit {
    
     this.showProgress();
     
+    const initialGender = '';
 
     this.defendantAndCaseForm = this.fb.group({
-
-      
         defendantname: new FormControl(),
         defendantrace: new FormControl(['', Validators.compose(
           [Validators.required, this.validateSelection]
         )]),
-        defendantgender: new FormControl(['', Validators.compose(
-          [Validators.required, this.validateSelection]
-        )]    ),
```

Comment:
https://stackoverflow.com/questions/42394999/why-should-i-use-validators-compose
I couldn't find anything official about validators.compose but apparently it is an older way to do things.